core/services/ocrcommon/data_source.go

```diff
@@ -33,9 +33,13 @@ type inMemoryDataSource struct {
 	chEnhancedTelemetry chan<- EnhancedTelemetryData
 }
 
+type saver interface {
```

Comment:
Should this be exported?
```suggestion
type Saver interface {
```

Reply:
🤦 Yes, it should -- the same interface is duplicated in a bunch of places that also import ocrcommon so exporting and deduping makes sense 👍